### PR TITLE
Issue#222 : Updated images and references for Meshery logo

### LINKED ISSUE #222: Replace the Logo of Layer5 with Meshery in config.yml workflow
### Description

Currently, we are using the Logo of Layer5 instead of Meshery. This repository is under the organization Meshery, so, there should be the image of Meshery.

**How to resolve this?**
- Go to .github and replace this welcome image with the new one: 

- Make sure to display the images conditionally, based on the theme selected by user.
Here's the sample code:

```
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/w.png">
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/b.png">
  <img alt="Meshery Logo" src="">
</picture>
```

New images are:

Meshery-celebration-Light
Meshery-celebration-Dark

- Download these images, and upload with an appropriate name. Make sure to use conditional rendering as described above.

- After uploading the image, and removing the previous image, go to .github/config.yml folder and update the reference to previous image to the new image in the following part:

```
firstPRMergeComment: >
  Thanks for your contribution to the Layer5 and Meshery community! :tada:
   
  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Layer5-celebration.png)
  &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; :star: Please leave a [star on the project](../stargazers). :smile:
```
Here, replace ```Congrats!``` with the code
```
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/w.png">
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/b.png">
  <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
</picture>
```
make sure to change the image sources with the path to image in the folder where you upload it.

- That's it.

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Ahmadshen**
**Description**
This PR fixes #222 

This PR updates the images and references in the repository to use the Meshery logo instead of the Layer5 logo. It also adds conditional rendering based on the user's selected theme. The following changes have been made:

I replaced the previous logo with the Meshery-celebration-Light and Meshery-celebration-Dark images.
Updated the <picture> element in the .github/config.yml file to use the appropriate image sources for light and dark themes.
Modified the firstPRMergeComment section in the config.yml file to display the new image using the <picture> element.

**Notes for Reviewers**
Please review the changes made to update the images and references in the repository. Ensure the new images are correctly displayed based on the user's selected theme. Test the changes locally and verify that they are functioning as expected.

I have followed the contribution conventions as mentioned in the community guidelines:

I have included a descriptive title for this PR, starting with [<component-name>].
I built and tested the changes before submitting this PR.
I have signed my commits.
Thank you for reviewing my contribution. I would greatly appreciate your feedback and guidance.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -17,7 +17,12 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery-nginx-sm/master/.github/welcome/Layer5-celebration.png)
+ <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/Ahmadshen/meshery-nginx-sm/raw/master/.github/welcome/Meshery-celebration-Dark.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://github.com/Ahmadshen/meshery-nginx-sm/raw/master/.github/welcome/Meshery-celebration-Light.svg">
+  <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="">
+ </picture>
```
Comment (thisiskaransgit) [inline .github/config.yml:24]: - We are using Raw github URLs for image targets
- Images should be sourced from the original repo, not your forked one

File: .github/config.yml [hunk 2]
```diff
@@ -17,7 +17,12 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
```
Comment (thisiskaransgit) [inline .github/config.yml:18]:
```suggestion
  Thanks for your contribution to Meshery! :tada:
```

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

**Review by Ahmadshen — COMMENTED** [review]
Hi, I have updated the changes as required. Please have a look at it and let me know your feedback. 
ALso, thank you for your feedback and correction, this means a lot! :)

File: .github/config.yml (hunk 1 above)
Comment (Ahmadshen) [reply .github/config.yml]: Thank you for your feedback, Karan. I have done changes as required. Please take a look and let me know. Thanks a lot

Comment (thisiskaransgit) [reply .github/config.yml]: @Ahmadshen, the URLs should be pointing at the master branch not a specific commit, if you are stuck take help from the similar PRs opened throughout the project

**Review by Ahmadshen — COMMENTED** [review]
Please have a look. Hopefully, this time, I am correct, and thank you for your feedback! : )

File: .github/config.yml (hunk 1 above)
Comment (Ahmadshen) [reply .github/config.yml]: @thisiskaransgit  Thank you so much for your constant feedback and suggestions, this means a lot to me. I have done changes as required and hopefully this time, it's good!

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

File: .github/config.yml [hunk 3]
```diff
@@ -15,9 +15,15 @@ newPRWelcomeComment: >
...
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Thanks for your contribution to the Layer5 community! :tada:
+  Thanks for your contribution to Meshery! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery-nginx-sm/master/.github/welcome/Layer5-celebration.png)
+ <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-nginx-sm/master/.github/welcome/Meshery-celebration-Dark.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery-nginx-sm/master/.github/welcome/Meshery-celebration-Light.svg">
+  <img alt="Shows a black logo in light color mode and a white one in dark color mode.">
```
Comment (anshgoyalevil) [inline .github/config.yml:23]: Please add a ```src="https://raw.githubusercontent.com/meshery/meshery-nginx-sm/master/.github/welcome/Meshery-celebration-Dark.svg"``` attribute here. This helps in adding a fallback image for browsers that do not support conditional rendering.

File: .github/config.yml [hunk 4]
```diff
@@ -15,9 +15,15 @@ newPRWelcomeComment: >
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Thanks for your contribution to the Layer5 community! :tada:
+  Thanks for your contribution to Meshery! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery-nginx-sm/master/.github/welcome/Layer5-celebration.png)
+ <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-nginx-sm/master/.github/welcome/Meshery-celebration-Dark.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery-nginx-sm/master/.github/welcome/Meshery-celebration-Light.svg">
```
Comment (anshgoyalevil) [inline .github/config.yml:22]: Interchange the dark and light with appropriate srcsets. The image with white text should be on a dark scheme, and vice versa.

**Review by anshgoyalevil — CHANGES_REQUESTED** [review]

File: .github/config.yml (hunk 3 above)
Comment (anshgoyalevil) [reply .github/config.yml:23]: This change needs to be done in the `````` tag here.

**Review by leecalcote — APPROVED** [review]
Thanks!

**Pull request merged**